### ISSUE TITLE
Purpose of parse_json.py?

### ISSUE THREAD

**Issue by geckon** [canismarko/dungeon-sheets#46]
Hi what is the purpose of `parse_json.py`? 

First of all, it uses an absolute path apparently only valid on @canismarko's computer:
```
INFILE = '/home/mwolf/Documents/dungeons_and_dragons/spells.json'
```

Also, there is a syntax error in the regex there:
```
components_re = re.compile('([VSM])?[, ]*([VSM])?[, ]*([VSM])?[, ]*'
                           '(?:\(([-a-zA-Z ,.0-9;\'-()]+)\))?')
```

**Comment by canismarko** [comment]
I'm guessing this is a remnant from when I first started making this thing, maybe I found a json file with all the spells so I used that instead of the proper spells.py file, or else I thought it made sense to store all the spells, etc, as json files instead of python.

**Comment by geckon** [comment]
In that case, do you want to remove it? I'd be happy to prepare a PR for this.

**Comment by canismarko** [comment]
Fixed: 82ae4dd63e19045884c55be6c0b2a41975f2ff56
